src/test/ui/consts/const_unsafe_unreachable_ub.rs

```diff
@@ -1,5 +1,4 @@
-// build-fail
-
+// error-pattern: evaluation of constant value failed
```

Comment:
This is not great, but arguably the old test already did not match in anything interesting in the error annotations either.